subprojects/ide/src/main/java/org/gradle/plugins/ide/idea/internal/IdeaScalaConfigurer.java

```diff
@@ -232,7 +237,7 @@ private VersionNumber findIdeaTargetVersion() {
         if (targetVersionString != null) {
             targetVersion = VersionNumber.parse(targetVersionString);
             if (targetVersion.equals(VersionNumber.UNKNOWN)) {
-                throw new GradleScriptException("String \'" + targetVersionString + "\' is not a valid value for IdeaModel.targetVersion.", null);
```

Comment:
Nitpick: this is an unrelated change.

Reply:
It's a warning in our current intellij settings (the escaping is unnecessary)

Reply:
No, it's me. I though you actually removed the single quotes. Ignore my comment, all good :)